backend/src/controllers/projects.controller.ts

```diff
@@ -1,12 +1,10 @@
 import { Request, Response } from 'express';
-import { Project, Task } from '../models/project';
+import { Project } from '../models/project';
 
 async function fetchProjectDetails(req: Request, res: Response) {
   try {
     const projectFilter = await Project.find();
-    projectFilter.map((project) => {
-      return project;
-    });
+    projectFilter.map((project) => (project.tasks = []));
```

Comment:
what is this line ? explain it please

Reply:
You said to not show the "Tasks" in "fetchProject"